config.json

```diff
@@ -0,0 +1,23 @@
+{
+  "database": {
+    "driver"   : "",
+    "host"     : "",
+    "user"     : "",
+    "password" : "",
+    "database" : "",
```

Comment:
This is one of the many things done in the other branch with the extra refactoring in. I've opened that up as a separate PR and closed this one so everyone is clear which is the one to test against :)